File: Algoritmos e logica de Programacao/Java/saida_de_dados.java
/* Saidad de dados em Java */
/* Exemplo da aula */


import java.util.Locale;

public class saida_de_dados {

	public static void main(String[] args) {
	    				
		Locale.setDefault(Locale.US);
		
		int idade;
		double salario;
		String nome;
		char genero;
		
		idade = 32;
		salario = 1100.5;
		nome = "Maria Jose";
		genero = 'F';
		
		System.out.println("A funcionaria " + nome + ", Sexo " + genero + ", ganha " + 	String.format("%.2f", salario) +
				" e tem " + idade + " anos.");
		
	}

}

File: Algoritmos e logica de Programacao/C/saida_de_dados.c
#include <stdio.h>

/*Saida de dados usando Placeholder*/


int main()
{

    int idade;
    double salario, altura;
    char genero;
    char nome[50];

    idade = 20;
    salario = 5800.5;
    genero = 'F';
    strcpy(nome, "Maria Silva");

    printf("A funcionaria %s, genero %c, ganha %.2lf e tem %d anos.", nome, genero, salario, idade);

    return 0 ;
}

